enterprise/frontend/src/embedding-sdk/hooks/private/use-load-question.ts

```diff
@@ -143,6 +146,17 @@ export function useLoadQuestion({
     [dispatch, originalQuestion],
   );
 
+  const updateVisualizationSettings = useCallback(
+    async (settings: VisualizationSettings) => {
+      if (!question || !shouldUpdateVisualizationSettings(settings)) {
+        return;
+      }
+
+      return updateQuestion(question.updateSettings(settings), { run: false });
```

Comment:
This matches the `onUpdateVisualizationSettings` redux action, but uses the context from the `useLoadQuestion` React hook instead.